applicationinsights-android/src/main/java/com/microsoft/applicationinsights/library/ApplicationInsights.java

```diff
@@ -311,19 +319,19 @@ public static void disableAutoPageViewTracking() {
      * {@link com.microsoft.applicationinsights.library.ApplicationInsights#start()}.
      */
     public static void enableAutoSessionManagement() {
-        if (!isRunning) {
-            InternalLogging.warn(TAG, "Could not set session management, because " +
+        if (!Util.isLifecycleTrackingAvailable()) {
```

Comment:
We are logging the same message multiple times in this class. Maybe it would be a good idea to put them elsewhere and reuse it.